http/src/main/resources/init-resources/jupyter_notebook_config.py

```diff
@@ -1,47 +0,0 @@
-# adapted from https://github.com/jupyter/docker-stacks/blob/master/base-notebook/jupyter_notebook_config.py
```

Comment:
Hm there might be some Leo code that's still referencing this file..

Reply:
See: 
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/reference.conf#L337

and
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/gce-init.sh#L308

Hmm maybe the file in Leo _is_ still used, and overwrites the version in terra-docker.